ibis/mapd/tests/test_client.py

```diff
@@ -38,6 +38,10 @@ def test_list_tables(con):
     assert len(con.list_tables(like='functional_alltypes')) == 1
 
 
+def test_sessionid_connection(session_con):
+    assert len(session_con.list_tables()) > 0
```

Comment:
This can be `assert session_con.list_tables()`

Reply:
:+1: